### PR TITLE
add getting current timezone

### PR DESCRIPTION
**Pull request by KazuCocoa**
As same as https://github.com/appium/WebDriverAgent/pull/184
(Will wait for 1.14.0 release)

### CODE REVIEW

File: test/unit/adb-commands-specs.js [hunk 1]
```diff
@@ -1202,4 +1203,16 @@ describe('adb commands', withMocks({adb, logcat, teen_process, net}, function (m
...
   });
+  describe('getCurrentTimeZone', function () {
+    it('should call shell with correct args', async function () {
+      mocks.adb.expects('shell')
+        .once().withExactArgs(['getprop', 'persist.sys.timezone'])
+        .returns(`Asia/Tokyo${EOL}`);
+      (await adb.getCurrentTimeZone()).should.equal('Asia/Tokyo');
+    });
+    it('should raise an error', async function () {
+      mocks.adb.expects('shell').throws(new Error());
```
Comment (mykola-mokhnach) [inline test/unit/adb-commands-specs.js]: the argument for `throws` can be skipped

File: lib/tools/adb-commands.js [hunk 2]
```diff
@@ -1724,4 +1724,20 @@ methods.performEditorAction = async function performEditorAction (action) {
...
 };
 
+/**
+ * Get tz database time zone formatted timezone
+ *
+ * @returns {string} TZ database Time Zones format
+ *
+ * @throws {error} If any exception is reported by adb shell.
+ */
+methods.getCurrentTimeZone = async function getCurrentTimeZone () {
```
Comment (mykola-mokhnach) [inline lib/tools/adb-commands.js]: why not just getTimeZone?

Comment (KazuCocoa) [reply lib/tools/adb-commands.js]: Ah, I implemented with iOS one...
(It had default/system/current timezone)

Comment (KazuCocoa) [reply lib/tools/adb-commands.js]: ah, no. I was confused with locale , haha

**Comment by KazuCocoa** [comment]
tests were passed, but archiving them failed.
will take a look at that after merging this...

**Pull request merged**